api/api.go

```diff
@@ -28,6 +28,9 @@ func Call(req *http.Request) (*http.Response, error) {
 	if req.URL.Scheme != scheme {
 		return nil, errors.New("invalid scheme")
 	}
+	if req.Header.Get("eventType") == "" {
```

Comment:
add reasons for the requirement of eventType in the header

Reply:
route result to wasm by the  eventType

Reply:
is callback necessary?